gems/laa_multi_step_forms/Gemfile.lock

```diff
@@ -113,7 +113,7 @@ GEM
       html-attributes-utils (~> 1.0.0, >= 1.0.0)
       pagy (~> 6.0)
       view_component (>= 3.9, < 3.10)
-    govuk_design_system_formbuilder (4.1.1)
+    govuk_design_system_formbuilder (5.0.0)
```

Comment:
eek, missed that when i bumped this gem for the app some time ago. I will let QA determine any impact. It cause no issues before....